src/probes/README.md

```diff
@@ -0,0 +1,53 @@
+# Probes
+
+The default probes provided along the telemetry client code are:
+
+* bertprobe: this probe reports on the Boot Error Region Table if such
+  entry is found in ```/sys/firmware/acpi/tables/BERT```.
+
+* crashprobe: this probe collects the core dump file from the kernel. This
+  probe is meant to be 'configured' in ```/proc/sys/kernel/core_pattern```.
+
+* hprobe: a simple probe that sends a keep alive message.
+
+* journalprobe: a probe for the journal.
+
+* oopsprobe: a probe to collect 'oops messages' when the kernel detects a
+  problem.
+
+* pstoreprobe: probe to collect messages left on pstore filesystem.
+
+
+### telem-record-gen
+
+Telemetry record generator is not exactly a probe, this tool can be used to
+implement probes i.e. using a bash script.
```

Comment:
I would describe telem-record-gen as a "general-purpose probe" for sending custom records on-the-fly. All other probes that come with telemetrics-client are special-purpose, since they are meant to collect a specific set of data and nothing else.